js/src/lib/components/InferenceWidget/widgets/ReinforcementLearningWidget/ReinforcementLearningWidget.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+	import type { WidgetProps } from "../../shared/types";
+	import { onMount } from "svelte";
+	import IconSpin from "../../../Icons/IconSpin.svelte";
+	import WidgetHeader from "../../shared/WidgetHeader/WidgetHeader.svelte";
+
+	export let apiToken: WidgetProps["apiToken"];
+	export let model: WidgetProps["model"];
+
+	enum Replay {
+		Loading,
+		Available,
+		Unavailable,
+	}
+
+	let replay = Replay.Loading;
+
+	function doesReplayExist() {
+		// check if repository has `replay.mp4` file
+		const replayUrl = `https://huggingface.co/${model.id}/resolve/main/output.mp4`;
```

Comment:
```suggestion
		const replayUrl = `/${model.id}/resolve/main/output.mp4`;
```
maybe no hardcode of domain name?

Reply:
oh indeed https://github.com/huggingface/hub-docs/pull/75/commits/4af46e9b338d45f2a9a594a9582dee43557f9854